service/src/main/kotlin/fi/espoo/evaka/shared/auth/AclQueries.kt

```diff
@@ -30,6 +30,20 @@ WHERE daycare_id = :daycareId
     .mapTo<DaycareAclRow>()
     .toList()
 
+fun Database.Read.hasDaycareAclRowForAnyUnit(employeeId: UUID, role: UserRole) = createQuery(
+    """
+        SELECT EXISTS(
+            SELECT * FROM daycare_acl
+            WHERE employee_id = :employeeId AND role = :role
+        )
+    """.trimIndent()
+)
+    .bind("employeeId", employeeId)
+    .bind("role", role)
+    .mapTo<Boolean>()
+    .first()
+    .let { it == true }
```

Comment:
Couldn't this be simplified to 
```kt
    .mapTo<Boolean>()
    .one()
```
?

Reply:
The idea is that `null` and `false` both map to `false`. Does `.one()` throw if the value is `null`?

Reply:
Ah, `Boolean!` seems to mean "`Boolean` or `Boolean?`", so `.one()` will do!